.github/blunderbuss.yml

```diff
@@ -13,218 +13,150 @@
+      - "api: functions"
+      - "api: run"
+      - "api: workflows"
+    to:
+      - GoogleCloudPlatform/torus-dpe
+  - labels:
+      - "api: asset"
+      - "api: cloudkms"
+      - "api: iam"
+      - "api: kms"
+      - "api: privateca"
+      - "api: recaptchaenterprise"
+      - "api: secretmanager"
+      - "api: securitycenter"
+    to:
+      - GoogleCloudPlatform/dee-infra
+  - labels:
+      - "api: clouderrorreporting"
+      - "api: cloudtrace"
+      - "api: container"
+      - "api: logging"
+      - "api: monitoring"
+      - "api: trace"
+    to:
+      - GoogleCloudPlatform/dee-platform-ops
+  - labels:
+      - "api: datacatalog"
+      - "api: datalabeling"
+      - "api: dialogflow"
+      - "api: enterpriseknowledgegraph"
+      - "api: language"
+      - "api: ml"
+      - "api: notebooks"
+      - "api: optimization"
+      - "api: speech"
+      - "api: talent"
+      - "api: texttospeech"
+      - "api: translate"
+      - "api: vision"
+    to:
+      - GoogleCloudPlatform/dee-data-ai
+
+  # DEE individuals
+  - labels:
+      - "api: batch"
+      - "api: compute"
+    to:
+      - m-strzelczyk
+  - labels:
+      - "api: dataflow"
+      - "api: people-and-planet-ai"
+    to:
+      - davidcavazos
+  - labels:
+      - "api: dataproc"
+      - "api: pubsub"
+      - "api: pubsublite"
+    to:
+      - anguillanneuf
+
+  # AppEco teams
+  - labels:
+      - "api: cloudsql"
+    to:
+      - GoogleCloudPlatform/infra-db-dpes
+  - labels:
+      - "api: bigtable"
+      - "api: datastore"
+      - "api: firestore"
+    to:
+      - GoogleCloudPlatform/cloud-native-db-dpes
+  - labels:
+      - "api: spanner"
+    to:
+      - GoogleCloudPlatform/api-spanner-python
+  - labels:
+      - "api: storage"
+    to:
+      - GoogleCloudPlatform/cloud-storage-dpes
+
+  # AppEco individuals
+  - labels:
+      - "api: bigquery"
+    to:
+      - shollyman
+  - labels:
+      - "api: composer"
+    to:
+      - leahecole
+      - rachael-ds
+      - rafalbiegacz
+  - labels:
+      - "api: datascienceonramp"
+    to:
+      - leahecole
+      - bradmiro
+  - labels:
+      - "api: contentwarehouse"
+      - "api: documentai"
+    to:
+      - holtskinner
+
+  # Self-service teams
+  - labels:
+      - "api: dlp"
+    to:
+      - GoogleCloudPlatform/googleapis-dlp
+  - labels:
+      - "api: functions"
+    to:
+      - GoogleCloudPlatform/functions-framework-google
+  - labels:
+      - "api: billingbudgets"
+      - "api: cloudbilling"
+    to:
+      - GoogleCloudPlatform/billing-samples-maintainers
+  - labels:
+      - "api: healthcare"
+    to:
+      - GoogleCloudPlatform/healthcare-life-sciences
+  - labels:
+      - "api: iot"
+      - "api: cloudiot"
+    to:
+      - GoogleCloudPlatform/api-iot
+  - labels:
+      - "api: retail"
+    to:
+      - GoogleCloudPlatform/cloud-retail-team
+
+  # Self-service individuals
+  - labels:
+      - "api: auth"
+    to:
+      - arithmetic1728
 
-assign_prs_by:
```

Comment:
Is it intended change to completely remove `assign_prs_by` section?

Reply:
I've made couple additional changes based on discussions around CODEOWNERS in [this commit](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9720/commits/fa06f7ae4736a5bef714edd77647c9de575213c9) (included in this PR).

The `assign_prs_by` section might require a longer discussion. I see following options:
 1) We keep the old one
 2) We remove it completely, which means that PRs will be auto assigned to GoogleCloudPlatform/python-samples-owners members only (might not be good for some of the self service teams)
 3) We duplicate the `assign_issues_by` section - hard to keep in sync + we may get people assigned who are unaware of the expectations related to being an assignee

Thoughts?

Reply:
I'm thinking option 3, with comments at the top of both sections (and maybe even the bottom of the file :) ) that both sections need to be updated.

Justification: this is only the assignee. If they see it, great - they have SME knowledge and will be flagged early in the process. If not, a Python docs samples reviewer will. And the process requires 2 reviewers anyway.